Allow the system header and library paths to be configured.
--- configure.orig	2007-03-06 18:21:34.000000000 -0600
+++ configure	2018-12-28 01:08:16.000000000 -0600
@@ -22,7 +22,6 @@
 if [ -d /usr/lib64 ]; then
     # SuSE x86_64
     LIBDIR=/usr/lib64
-    echo "using $LIBDIR for LIBDIR" 1>&2
 fi
 
 # largefile support OK
@@ -149,6 +148,12 @@
     --force-sdbm)
 	FORCE_SDBM=true
 	;;
+    --incdir=*)
+	INCDIR=`echo "$1" | sed 's/--incdir=//'`
+	;;
+    --libdir=*)
+	LIBDIR=`echo "$1" | sed 's/--libdir=//'`
+	;;
     --longlong)
 	LONGLONG=true
 	;;
@@ -232,6 +237,10 @@
 	--force-memmove		--no-dlopen
 	--no-dynlib		--no-largefile
 
+System headers and libraries;
+	--incdir=PATH		[default: /usr/include]
+	--libdir=PATH		[default: /usr/lib or /usr/lib64]
+
 Install location;
 	--prefix=PATH		[default: /usr/local]
 	--bindir=PATH		[default: PREFIX/bin]
@@ -1148,11 +1157,9 @@
 	case "${OS}${OSREL}" in
 	Darwin*)
 	    # darwin dlopen() support in cctools/libdyld??
-	    if [ -f /usr/lib/libdl.dylib ]; then
+	    if [ -f $LIBDIR/libdl.dylib ]; then
 		# Darwin 7.9 (OS X 10.3) and later??
 		echo 'ADD_LDFLAGS([-ldl])'	>> $CONFIG_M4
-	    elif [ -f /sw/lib/libdl.dylib ]; then
-		echo 'ADD_LDFLAGS([-L/sw/lib -ldl])' >> $CONFIG_M4
 	    fi
 	    TRY_UNDERSCORE=true
 	    SNOLIB_FILE=snolib.bundle
@@ -1490,7 +1497,7 @@
 	echo '#define HAVE_NDBM_H'		>> $CONFIG_H
 	# see if ndbm.h is Berkeley DB wrapper
 	if grep 'Margo Seltzer' $INCDIR/ndbm.h > /dev/null &&
-	    [ -f /usr/lib/libdb.a -o -f /usr/lib/libdb.so ]; then
+	    [ -f $LIBDIR/libdb.a -o -f $LIBDIR/libdb.so ]; then
 	    # RH5.2 has Berkeley DB ndbm.h, but seperate libdb.a
 	    # (BSD systems have BDB in libc)
 	    echo "using libdb for DBM support" 1>&2
